src/backend/TrafficCourts/TrafficCourts.Staff.Service.Test/Controllers/DisputeControllerTest.cs

```diff
@@ -20,10 +21,11 @@ public async void TestGetDisputes200Result()
         // Mock the IDisputeService to return a couple Disputes, confirm controller returns them.
 
         // Arrange
+        Guid id = Guid.NewGuid();
         Dispute dispute1 = new();
-        dispute1.Id = 1;
+        dispute1.Id = id;

```

Comment:
Or use different ids for the different records
```suggestion
        dispute1.Id = Guid.NewGuid();
        dispute2.Id = Guid.NewGuid();
```